src/pages/account/logins/[type]/validation.tsx

```diff
@@ -0,0 +1,102 @@
+import { HttpStatus } from "@fewlines/fwl-web";
```

Comment:
You can move the form into `/display/IdentityValidationForm`. We won't do the business logic for now, it will be its own PR, with the validation workflow implementation.

Reply:
OK fixed! 4a2f81f